packages/mapviewer/src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -32,28 +32,51 @@ const sanitizedFeatureDataEntries = computed(() => {
     if (hasFeatureStringData.value || !feature?.data) {
         return []
     }
-    return Object.entries(feature.data)
-        .filter(([_, value]) => value) // filtering out null values
+    return Object.entries(feature?.data)
+        .filter(([_, value]) => value) // Filtering out null values
         .map(([key, value]) => [
             key,
             sanitizeHtml(value, key === 'description'),
-            getIframeHosts(value),
+            key === 'description' ? getIframeHosts(value) : [],
         ])
 })
 function sanitizeHtml(htmlText, withIframe = false) {
+    const blockedExternalContentString = t('blocked_external_content')
+
+    // Replacing possibly malicious code with `blockedExternalContentString` instead of removing it
+    function handleNode(node, attribute) {
+        try {
+            const url = new URL(node.getAttribute(attribute))
+            const ext = url.pathname.split('.').pop().toLowerCase()
+            if (BLOCKED_EXTENSIONS.includes(ext)) {
+                node.outerHTML = blockedExternalContentString
+                return true
+            }
+        } catch (error) {
+            log.error(error)
+            node.outerHTML = blockedExternalContentString
+            return true
+        }
+        return false
+    }
```

Comment:
you can remove the `return` values here, no more needed.

can you also contextualize a bit the `log.error` by giving some error message before the error object? Will be way easier to find in the code when we see the error in the JS console later.